src/libraries/System.Collections.Specialized/src/Resources/Strings.resx

```diff
@@ -96,4 +96,10 @@
   <data name="Serialization_InvalidOnDeser" xml:space="preserve">
     <value>OnDeserialization method was called while the object was not being deserialized.</value>
   </data>
+  <data name="OrderedDictionaryKeys_ReadOnly" xml:space="preserve">
```

Comment:
I think it might be possible to reuse the wording found in the equivalent error messages in generic collections:

https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/System.Collections/src/Resources/Strings.resx#L102-L107